src/reader/TranslatableText.sc.js

```diff
@@ -10,9 +10,10 @@ import {
 
 const TranslatableText = styled.div`
   z-tag {
-    padding-right: 0.3em;
+    white-space: break-spaces;
     cursor: pointer;
-    /* margin-right: 1em; */
+    /* margin-right: 1em; 
```

Comment:
I just realized that I've left these commented lines in there and the good practice would be to not allow commented out code.